### ISSUE TITLE
Safari dropdown fields not working anymore

### ISSUE THREAD

**Issue by martinantonmueller** [acdh-oeaw/apis-core#293]
It's strange, I believe I created an issue today, but can’t find it. On the current MacOs/Safari the dropdown-fields with possibility to enter text cannot be accessed. 
Reason seems to be the plugin: autocomplete_light.min.js that cannot be found, see screen-shot below

**Comment by skurzinz** [comment]
You may be refering to the closed #292 :)

**Comment by martinantonmueller** [comment]
No, that was because PMB was moved to cluster. But afterwards it still didn't work. ;-)

**Comment by martinantonmueller** [comment]
The bug exists in Safari 15.1 as well.

**Comment by sennierer** [comment]
That is a DAL issue according to this issue https://github.com/yourlabs/django-autocomplete-light/issues/1226 upgrading to latest version should fix it. I'll check and update.

**Comment by sennierer** [comment]
made a mistake in the commit message. This c06d06433b6a119b1f88635edfb20c0604389e5f should resolve the autocomplete bug

**Comment by martinantonmueller** [comment]
This is still (again) a bug that includes IOS and iPadOS. While on the Mac switching to Firefox or Chrome solves it, on the mobile devices all browser have the bug.
